<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblPoza1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../animalBd/test_set/test_set/cats/cat.4008.jpg</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rdbLines">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lines</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbBlur">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Blur</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbCrop">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cropping</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbEdge">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edge Detection</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbTresh">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Thresholding</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbOglinda">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rotire</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbResize">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Resizing</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbGrey">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HoloLens MDL2 Assets</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Color Spaces</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblTxtPoza1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCautaPoza">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Introduce Poza</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetare">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resetare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnValideaza">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Valideaza Alegeri</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPoza2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../animalBd/test_set/test_set/cats/cat.4008.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblPoza3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>110</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../animalBd/test_set/test_set/cats/cat.4008.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblTxtPoza2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTxtPoza3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTxtPoza4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTxtPoza5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>410</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPoza6">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>450</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../animalBd/test_set/test_set/cats/cat.4008.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblPoza5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../animalBd/test_set/test_set/cats/cat.4008.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblTxtPoza6">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>410</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPoza4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../animalBd/test_set/test_set/cats/cat.4008.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStanga">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotatie Stanga</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDreapta">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotatie Dreapta</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcent">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInc">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Incearca</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtX">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>760</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtY">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>760</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtZ">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>760</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtW">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>760</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDate">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Incearca</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDimensiune">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>810</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1325</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblProcent</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>852</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
